lifecycle/test/test_lifecycle.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import unittest
+
+import launch
+import launch.actions
+import launch.event_handlers.on_process_start
+
+
```

Comment:
nit: remove extra empty line

Reply:
Removed in https://github.com/ros2/demos/pull/336/commits/17757acdcc6c41779d87af75673d178de071e12d.